problem-sets-competitive-programming.md

```diff
@@ -34,6 +35,12 @@
 * [URI Online Judge](https://www.urionlinejudge.com.br/judge/en/login)
 
 
+### ctf
```

Comment:
use the full heading (need to change the link to correspond

Reply:
okay , makes sense.

Reply:
whT bout the ctf links in security?

Reply:
those links were more of practice and pentesting labs arena which i added in competitive programming section.

Reply:
I will add more resources in future as I myself have started doing ctf few months back.

Reply:
Great initiative !!!